ansible/environments.yml

```diff
@@ -40,7 +40,8 @@
           node_flavor: c1.medium
           image_name: "d324da4b-b827-44fb-8331-bfad41a28aca"
       tags:
-        - integration-env
```

Comment:
I did not check the `-env` suffix.

Reply:
I don't find any usage, it's primary purpose was manual runs. I changed it for consistency - "the ansible way" of tags (no names, only values).